ui/panels-plugin/src/plugins/stat-chart/stat-chart-model.ts

```diff
@@ -26,6 +27,7 @@ export interface StatChartOptions {
   unit: UnitOptions;
   thresholds?: ThresholdOptions;
   sparkline?: StatChartSparklineOptions;
+  fontSize?: FontSizeOptions;
```

Comment:
Would it be worth distinguishing that this is `value_font_size`? We currently don't display any other labels but I could see that being added in the future similar to Grafana ([ex](https://play.grafana.org/d/Zb3f4veGk/2-stats?orgId=1)). 

Also can you update the CUE schema for StatChart [here](https://github.com/perses/perses/blob/main/schemas/panels/stat/stat.cue), could be something like: 
```
calculation: common.#calculation
unit?:  common.#unit
thresholds?:  common.#thresholds
sparkline?:  #sparkline
value_font_size?: number
```

Then some of the make commands can be useful like: 
- make fmt
- make cue-eval
- make cue-test

Reply:
Yeah it can't hurt to be more specific. I changed it to `value_font_size` and updated the cue file!